tests/common/plugins/sanity_check/checks.py

```diff
@@ -75,6 +75,7 @@ def _find_down_ports(dut, phy_interfaces, ip_interfaces):
 def check_interfaces(duthosts):
     def _check(*args, **kwargs):
         result = parallel_run(_check_interfaces_on_dut, args, kwargs, duthosts.frontend_nodes, timeout=600)
+        logger.debug("Result of check_interfaces: {}".format(result))
```

Comment:
I think the logs added in this file are unnecessary. They should can be logged by the change you made to `sanity_check/__init__.py`, right?

Reply:
Yes, you are right, I delete these.